Automattic/jetpack
# Storybook: remove knobs from stories

When we first created the Storybook environment, we used Knobs, which is a storybook extension. But there is a built in (actually called essential extension) called Controls that automatically reads propTypes and looks and works much better.

This PR implements `propTypes` and use Controls in the components stories - and removes the knobs dependency from Storybook

#### Changes proposed in this Pull Request:

* Removes knobs dependency from Storybook

#### Jetpack product discussion


1200295095943283-as-1201236438619239

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Build all js-packages
* Launch Story book `projects/js-packages/storybook && pnpm run storybook:dev`
* Browse all the components and confirm they look fine both in Canvas and Docs views
* Make sure the props are properly documented
* Make sure control works
* Look at the color picker for the logoColor and see how beautiful it is

Also make sure everything still works in the real world

* Visit the Jetpack Backup page, make sure header and footer still look the same
